### ISSUE TITLE
[React] Datepicker for React

### ISSUE THREAD

**Issue by johannesoak** [johannesoak/React-Design-Date-Picker#1]
### There is a bunch of stuff on my to-do list to make this date picker for React more complete.
Feel free to make a pull request and fix anything of these tasks.

**To-Do**
- Design the draggable element.
- Create a mobile design.
- Create and verify Responsive design.
- Add main color and sub-color options.
- Add accessability keyboard support.
- Validate that it's working as expected in different timezones.
- Add Framer animations.
- Create a infinity scroll layout.
- Connect outer setHooks with internal date values.
- Fix minor design issues on Compact design.
- Create tests using Jest framework.
- Add a time input.

**Bug to be fixed**
Sets date when selecting month or year in the picker.
You can re-produce the bug if you open the date picker, click on the month, choose a month and then a date is being picked on that page even though the date should be the active date.
Eg. active date is 1 January and you change it to February - Then the active date is being highlighted as 1 February.

**Comment by RitikPandey1** [comment]
Hi @johannesoak ,
I would like to work on this

**Comment by johannesoak** [comment]
I think you just can fork this repo, make code changes and then do a pull request @RitikPandey1

**Comment by RitikPandey1** [comment]
Okay that's great

**Comment by JuWeber99** [comment]
> Okay that's great

You doin something specific? If yes I whould love to share a few tasks :). What do you think?

**Comment by RitikPandey1** [comment]
> > Okay that's great
> 
> You doin something specific? If yes I whould love to share a few tasks :). What do you think?

Yes, I am working on bug

**Comment by JuWeber99** [comment]
> > > Okay that's great
> > 
> > 
> > You doin something specific? If yes I whould love to share a few tasks :). What do you think?
> 
> Yes, I am working on the bug and after it I'd like to work on infinite scroll feature.

can i contact u soon so we work it off together?

**Comment by RitikPandey1** [comment]
> > > > Okay that's great
> > > 
> > > 
> > > You doin something specific? If yes I whould love to share a few tasks :). What do you think?
> > 
> > 
> > Yes, I am working on the bug and after it I'd like to work on infinite scroll feature.
> 
> can i contact u soon so we work it off together?

I have fixed the bug and open the PR , you could work on other features @JuWeber99

**Comment by johannesoak** [comment]
Removed bug since it's fixed by @RitikPandey1

**Comment by vinlegend1** [comment]
Hey, is this still available to work on? I think I would like to add testing and draggable elements

**Comment by johannesoak** [comment]
@vinlegend1 Yes, go ahead! 👍  I don't know anyone who works at those task - Atleast what I've heard of and no pull request :)

**Comment by mihmilicio** [comment]
I'll start to work on the tests for this...

**Comment by mihmilicio** [comment]
@vinlegend1 are you working on testing? If not, I'd like to take it

**Comment by vinlegend1** [comment]
> @vinlegend1 are you working on testing? If not, I'd like to take it

Go ahead, I had a change of plans and am working on a different project, woops.

**Comment by ahbab-zaman** [comment]
I would like to contribute

**Comment by gmjishara** [comment]
I would like to work on this issues

**Comment by ankit7847** [comment]
Hi @johannesoak ,
I would like to work on this

**Comment by spe9110** [comment]
I would like to contribute
